apps/vite/src/components/calendar/EventList.tsx

```
@@ -1,26 +1,71 @@
-import React from 'react'
-import UpcomingEvent from './UpcomingEvent'
+import React from "react"
+import UpcomingEvent from "./UpcomingEvent"
+import { EventDetail, NewCalendarData } from "@miti/types"
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)